SU-HSDO/stanford_mrc
# MRCD8-281 fix image floating issue on horizontal postcard pattern

# READY FOR REVIEW

# Summary
- Added max-width to container of horizontal postcard image on everything above mobile breakpoint

# Needed By (Date)
- End of sprint (3/15)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the horizontal postcards on the Event Series view page looks right (image floated left and text right until mobile, which is when the image extends FW inside of its container)

# Affected Projects or Products
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-281

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
